src/main/java/org/jabref/gui/openoffice/DetectOpenOfficeInstallation.java

```diff
@@ -51,33 +46,34 @@ public DetectOpenOfficeInstallation(JDialog parent, OpenOfficePreferences prefer
         if (this.checkAutoDetectedPaths(preferences)) {
             future.complete(true);
         } else {
-            init();
+            DefaultTaskExecutor.runInJavaFXThread(() -> init());
```

Comment:
Is `runInJavaFXThread` really needed here? If yes, please move it to the caller of `isInstalled`.

Reply:
I refactored this and splitted it up in sub tasks